recipes/nlohmann_json/all/conanfile.py

```diff
@@ -38,11 +40,15 @@ def build(self):
         cmake.build()
 
     def package(self):
-        self.copy(pattern="LICENSE*", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="LICENSE*", dst="licenses",
+                  src=self._source_subfolder)
         cmake = self._configure_cmake()
         cmake.install()
         tools.rmdir(os.path.join(self.package_folder, "lib"))
         try:
             os.remove(os.path.join(self.package_folder, "nlohmann_json.natvis"))
         except FileNotFoundError:
             pass
+
+    def package_id(self):
+        self.info.header_only()
```

Comment:
```suggestion
    def package_id(self):
        self.info.settings.clear()
```

`multiple_headers` option produces two different packages I guess

Reply:
Good catch!